standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -1418,13 +1453,17 @@ public void commitTxn(CommitTxnRequest rqst) throws NoSuchTxnException, TxnAbort
 
     boolean isReplayedReplTxn = TxnType.REPL_CREATED.equals(rqst.getTxn_type());
     boolean isHiveReplTxn = rqst.isSetReplPolicy() && TxnType.DEFAULT.equals(rqst.getTxn_type());
+    TransactionStatus status = null;
```

Comment:
where is it being used?

Reply:
Fixed. Not storing the result of `jdbcTemplate.getTransactionWithinRetryContext()` as it is not used explicitly at all.